### ISSUE TITLE
Compatibility with the SPL + Metaplex Standard

### ISSUE THREAD

**Issue by vpontis** [glow-xyz/nftoken#19]
It would be nice to have a migration path from the existing standard to the new standard.

**Comment by crypt0miester** [comment]
upgrade to "nftoken" and get half of your money. lol

I like the idea. 100s of systems/platforms out there would need to change a lot of codes though. 

can a program change another program's account not owned by the latter program?

**Comment by vpontis** [comment]
> can a program change another program's account not owned by the latter program?

I don't think so. I think it'll be tough to migrate existing NFTs and I'm not focused on compatibility for launch.
